data/hai/hai news.txt

```diff
@@ -474,7 +474,7 @@ news "Unfettered Regret"
 		word
 			`"`
 		word
-			"At first I thought that joining the Unfettered would be fun. Now I can't go back or my crewmembers will beat me."
+			"At first I thought that joining the Unfettered would be fun. Now I can't go back or my crew members will beat me."
```

Comment:
```suggestion
			"At first I thought that joining the Unfettered would be fun. Now I can't go back or my crewmembers will beat me."
```

"[crewmember](https://en.wiktionary.org/wiki/crewmember)" is actually fine as a gender neutral form of "crewman".